xdotool/functions

```diff
@@ -124,7 +133,7 @@ function store-window-id() {
     # the terminal used to type the command, as this function is run as a prexec
     # hook.
     if [[ -z "$WINDOWID" || "$always_check" == 'yes' ]]; then
-        zstyle ':notify:*' window-id "$(xdotool getwindowfocus)"
+       # zstyle ':notify:*' window-id "$(xdotool getwindowfocus)"
```

Comment:
i think we need this, no?

Reply:
Oh sorry - i only need this in wayland enviroments (like gnome, sway, ...).
That should not be here in this PR.